### PR TITLE
Add Haberdasher to Dockerfile

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Add haberdasher (wrapper for message logging) to the Dockerfile

## Type of change

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,6 +44,12 @@ COPY --from=builder /rules-content/content/ /rules-content
 # copy tutorial/fake rule to external rules to be hit by all reports
 COPY rules/tutorial/content/ /rules-content/external/rules
 
+RUN curl -L -o /usr/bin/haberdasher \
```
Comment (quarckster) [inline Dockerfile:47]: Let's not use `latest` release. Please specify the exact version/

**Review by quarckster — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (epapbak) [reply Dockerfile:47]: Sure thing. I'll set it to v0.1.3, which is the latest right now.

**Review by quarckster — APPROVED** [review]

**Review by joselsegura — COMMENTED** [review]
Is this the intended way to install `haberdasher`, curling it from GitHub repository? It seems a bit weird

**Comment by epapbak** [comment]
> Is this the intended way to install `haberdasher`, curling it from GitHub repository? It seems a bit weird

This is the way indeed. See https://github.com/RedHatInsights/haberdasher/blob/master/README.md#adding-it-to-your-dockerfile

**Review by dpensi — APPROVED** [review]

**Review by tisnik — APPROVED** [review]
LGTM, but the installation part is strange :)

**Review by joselsegura — APPROVED** [review]

**Comment by joselsegura** [comment]
> > Is this the intended way to install `haberdasher`, curling it from GitHub repository? It seems a bit weird
> 
> This is the way indeed. See https://github.com/RedHatInsights/haberdasher/blob/master/README.md#adding-it-to-your-dockerfile

Well, I don't like it, but if it is the intended way, go ahead LGTM

**Pull request merged**